src/com/company/Sudoku.java

```diff
@@ -0,0 +1,380 @@
+package com.company;
+
+import javax.swing.*;
```

Comment:
kurcze jakim cudem tutaj swinga zaimportowałeś to ja nawet nie wiem
niepotrzebny import w każdym razie
każdy import to zaciągnięcie jakiejś biblioteki do naszego programu, swing jest na tyle dużym projektem, że waży pewnie z 10 razy więcej niż nasz